extension/content/utils/api.js

```diff
@@ -8,7 +8,7 @@ export default class API {
     throw new Error("getBaseURL() needs to be implemented");
   }
 
-  async request({ url, extraHeaders = {}, version = 3, ...options }) {
+  async request({ url, version, extraHeaders = {}, ...options }) {
```

Comment:
Typescript is grumpy because you removed the default value here, but the Experimenter call site at `extension/content/utils/experimenterApi.js` doesn't pass. It's not effectively required since the ExperimenterAPI overrides `getBaseUrl` and provides a default value their, but Typescript can't know that (and I didn't when I read it).

One way you could fix this is to sort out whether this should be required or not. You could declare it as specifically optional like this:

```suggestion
  /**
   * @param {Object} args
   * @param {string} args.url
   * @param {?number} args.version
   * @param {?Object} args.extraHeaders
   * @param {?string} args.body
   */
  async request({ url, version, extraHeaders = {}, ...options }) {
```